packages/apps/spaces/ui/media/icons/EventIllLog.tsx

```diff
@@ -0,0 +1,128 @@
+import { Icon, IconProps } from '@ui/media/Icon';
```

Comment:
I would not import this as an icon looks like a heavy graphic image.

Reply:
this file is removed